Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -404,107 +352,72 @@ void CGPathAddArc(CGMutablePathRef path,
     }
 
     // Call the internal function for breaking the arcs down into smaller segments
-    _CGPathAddArc(path, m, x, y, radius, startAngle, endAngle, clockwise);
+    _CGPathAddArc(path, transform, x, y, radius, startAngle, endAngle, clockwise);
 }
 
 /**
  @Status Interoperable
 */
-void CGPathMoveToPoint(CGMutablePathRef path, const CGAffineTransform* m, float x, float y) {
-    if (path == NULL) {
-        return;
-    }
-
-    if (m) {
-        CGPoint pt;
-
-        pt.x = x;
-        pt.y = y;
-
-        pt = CGPointApplyAffineTransform(pt, *m);
-
-        x = pt.x;
-        y = pt.y;
+void CGPathMoveToPoint(CGMutablePathRef path, const CGAffineTransform* transform, float x, float y) {
```

Comment:
Good luck? Haha. That seems like something explicitly bad to do that sounds like undefined behavior. Here we wouldn't get any particular change since the current point and starting point are only used in tandem with adding actual geometry figures.

---
In reply to: [83932143](https://github.com/Microsoft/WinObjC/pull/1138#discussion_r83932143) [](ancestors = 83932143)